packages/qwik/src/core/v2/client/vnode-diff.ts

```diff
@@ -567,9 +573,22 @@ export const vnode_diff = (container: ClientContainer, jsxNode: JSXOutput, vStar
       const element = vnode_getNode(vnode) as QElement2;
       if (!element.qDispatchEvent) {
         element.qDispatchEvent = (event: Event) => {
-          const eventName = event.type;
+          let eventName = event.type;
```

Comment:
I don't think this is right. See comment bellow about it having to start with `:`